test/crd_checks.go

```diff
@@ -30,7 +30,7 @@ import (
 
 const (
 	interval = 1 * time.Second
-	timeout  = 5 * time.Minute
+	timeout  = 10 * time.Minute // TODO: timeout should be configurable via go test timeout
```

Comment:
New pipelinerun e2e test includes 4 tasks and this takes upto 6min (on my gke) cluster. I wanted to add timeout to be on safer side so increased it upto 10min but it is very specific to that e2e test. So probably I will just update timeout for that test instead of all crds check timeout